src/osd/PG.cc

```diff
@@ -3569,22 +3569,6 @@ void PG::sub_op_scrub_unreserve(OpRequestRef op)
   clear_scrub_reserved();
 }
 
-void PG::sub_op_scrub_stop(OpRequestRef op)
```

Comment:
@tchaikov no callers for this now.. but is this something we might want later?  e.g., if 'noscrub' is set on the cluster or something?